tests/pageObjects/AddProduct.js

```diff
@@ -0,0 +1,39 @@
+class AddProduct{
+    constructor(){
+        this.accountSelector ="//button[@id='account-button']";
+        this.adminPanelSelector = "//a[contains(text(), 'Admin Panel')]";
+        this.sidebarProductSelector = "//a[@href='#sidebar-products']";
+        this.menuItemProductSelector = "//li[contains(@class,'sidebar-menu-item ')] /a[@href = '/admin/products']";
+        this.newProductButtonSelector = "//div[contains(@class, 'page-actions')]/a[@id = 'admin_new_product']";
+        this.productNameSelector = "//input[@id='product_name']";
+        this.skuSelector = "//input[@id='product_sku']";
+        this.prototypeSelectOptionSelector = "//span[@id='select2-product_prototype_id-container']";
+        this.prototypeTextFieldSelector = "//input[@class='select2-search__field']";
+        this.productPriceSelector = "//input[@id='product_price']";
+        this.shippingCategorySelector = "//span[@aria-labelledby='select2-product_shipping_category_id-container']";
+        this.shippingCategoryDigitalSelector = "//li[text()='Digital']";
+        this.createButtonSelector = "//button[@type='submit']";
+        this.productSelector= "//a[text()='%s']";
+    }
+
+    async navigateToNewProductPage(){
```

Comment:
Use this function just to navigate to the products page.
```suggestion
    async navigateToProductsPage(){
```